src/ape/utils/misc.py

```diff
@@ -490,9 +490,33 @@ def is_zero_hex(address: str) -> bool:
         return False
 
 
+def dict_defaults(mapping: Dict[str, Any], defaults: Dict[str, Any], depth=0):
```

Comment:
I don't see this method being used, is it still needed?

Reply:
Speculative future use.  I'll axe it.  Just didn't want to write it again.